File: CS/MvcDashboardDataSources/Data/Category.cs
using DevExpress.Xpo;

namespace MvcDashboardDataSources {
    [Persistent("Categories"), DeferredDeletion(false)]
    public class Category : XPCustomObject
    {
        int categoryId;
        string categoryName;
        string description;
        [Key]
        public int CategoryID
        {
            get { return categoryId; }
            set { SetPropertyValue<int>("CategoryID", ref categoryId, value); }
        }
        public string CategoryName
        {
            get { return categoryName; }
            set { SetPropertyValue<string>("CategoryName", ref categoryName, value); }
        }
        public string Description
        {
            get { return description; }
            set { SetPropertyValue<string>("Description", ref description, value); }
        }
    }
}

File: VB/MvcDashboardDataSources/Data/Category.vb
Imports DevExpress.Xpo

Namespace MvcDashboardDataSources
	<Persistent("Categories"), DeferredDeletion(False)>
	Public Class Category
		Inherits XPCustomObject

'INSTANT VB NOTE: The field categoryId was renamed since Visual Basic does not allow fields to have the same name as other class members:
		Private categoryId_Renamed As Integer
'INSTANT VB NOTE: The field categoryName was renamed since Visual Basic does not allow fields to have the same name as other class members:
		Private categoryName_Renamed As String
'INSTANT VB NOTE: The field description was renamed since Visual Basic does not allow fields to have the same name as other class members:
		Private description_Renamed As String
		<Key>
		Public Property CategoryID() As Integer
			Get
				Return categoryId_Renamed
			End Get
			Set(ByVal value As Integer)
				SetPropertyValue(Of Integer)("CategoryID", categoryId_Renamed, value)
			End Set
		End Property
		Public Property CategoryName() As String
			Get
				Return categoryName_Renamed
			End Get
			Set(ByVal value As String)
				SetPropertyValue(Of String)("CategoryName", categoryName_Renamed, value)
			End Set
		End Property
		Public Property Description() As String
			Get
				Return description_Renamed
			End Get
			Set(ByVal value As String)
				SetPropertyValue(Of String)("Description", description_Renamed, value)
			End Set
		End Property
	End Class
End Namespace